source/docs/software/can-devices/can-addressing.rst

```diff
@@ -189,17 +189,22 @@ For CAN Nodes to be accepted for use in the FRC System, they must:
 Universal Heartbeat
 -------------------
 
-The roboRIO provides a universal CAN heartbeat that any device on the bus can listen and react to. This heartbeat is sent every 20ms.
-The heartbeat has a full CAN ID of 0x01011840 (which is the NI Manufacturer ID, RobotController type, Device ID 0 and API ID 0x062).
-It is an 8 byte CAN packet. The important byte in here is byte 5 (index 4). The layout is the following bitfield.
-
-RedAlliance : 1
-Enabled : 1
-Autonomous : 1
-Test : 1
-WatchdogEnabled : 1
-Reserved : 3
-
-The flag to watch for is WatchdogEnabled. If that flag is set, that means motor controllers are enabled.
+The roboRIO provides a universal CAN heartbeat that any device on the bus can listen and react to. This heartbeat is sent every 20ms. The heartbeat has a full CAN ID of ``0x01011840`` (which is the NI Manufacturer ID, RobotController type, Device ID 0 and API ID ``0x062``). It is an 8 byte CAN packet. The important byte in here is byte 5 (index 4). The layout is the following bitfield.
+
++-----------------+-------+
+| Description     | Width |
++=================+=======+
+| RedAlliance     | 1     |
++-----------------+-------+
+| Enabled         | 1     |
++-----------------+-------+
+| Autonomous      | 1     |
```

Comment:
You forgot to put the test bit in the table.

Reply:
Also, I think this part specifically should stay how I had it. How I had it is how a bit field is designed in C, so that's really how people that understand bitfields would expect it to look.

Reply:
Bitfields aren't shown in a table normally? You basically had a table previously... just without the table formatting.

Reply:
Not normally no. What I had is the actual C syntax for a bitfield which is pretty ubiquitous.

Reply:
Are the previous examples in this article bitfields? Should they no longer be tables?

Reply:
I also do see other examples of bitfields in tables from a brief google search

Reply:
The previous items are not bitfields, so those should be tables still. I guess a table is fine. But you do need to add in the item you missed. Otherwise its just wrong.